### PR TITLE
feat(seo): Schema changes

### PR DESCRIPTION
**Pull request by nandap4790**

### CODE REVIEW

**Comment by nandap4790** [comment]
1. Add article section for the article schema.
2. Add article body(text-content) for the article schema.
3. Add story url as the mainEntity id, based on the params sent from the app(param can be removed once bq team confirms to add the storyurl as well- currently publisher url is being used).
4. Add story headline as the name.
5. Add width and height of the publisher logo with the organization schema

File: src/structured-data/structured-data-tags.js [hunk 1]
```diff
@@ -46,15 +49,36 @@ function authorData(authors) {
   return (authors || []).map(author => getSchemaPerson(author.name));
 }
 
+function getTextElementOfStory(story) {
+  if(story && story.cards) {
+    return story.cards.map((card) => {
```
Comment (Jeevan-Kishore) [inline src/structured-data/structured-data-tags.js]:
```
story.cards.map(card => card['story-elements'].filter(element => element.type === 'text');
```

File: src/structured-data/structured-data-tags.js [hunk 2]
```diff
@@ -46,15 +49,36 @@ function authorData(authors) {
...
-
+  const articleSection = story.sections && story.sections[0]['display-name'];
   return Object.assign({}, generateCommonData(structuredData, story, publisherConfig), {
     "author": authorData(authors),
     "keywords": metaKeywords,
+    "articleSection": articleSection,
+    "articleBody": (Object.keys(story).length > 0 && getCompleteText(story)) || '',
     "dateCreated": stripMillisecondsFromTime(new Date(story['created-at'])),
     "dateModified": stripMillisecondsFromTime(new Date(story['updated-at'])),
+    "name": (Object.keys(story).length > 0 && story.headline) || ''
```
Comment (Jeevan-Kishore) [inline src/structured-data/structured-data-tags.js]: make this a const

File: src/structured-data/structured-data-tags.js [hunk 3]
```diff
@@ -46,15 +49,36 @@ function authorData(authors) {
...
+  })
+  const completeCardText = joinTextOfSingleCard.join('.');
+  return completeCardText;
+}
+
 function generateArticleData (structuredData = {}, story = {}, publisherConfig = {}){
   const metaKeywords = story.seo && story.seo['meta-keywords'] || [];
   const authors = story.authors && story.authors.length !== 0 ? story.authors : [{name: story["author-name"] || ""}];
-
+  const articleSection = story.sections && story.sections[0]['display-name'];
```
Comment (Jeevan-Kishore) [inline src/structured-data/structured-data-tags.js]: use get

File: src/structured-data/structured-data-tags.js [hunk 4]
```diff
@@ -115,9 +139,12 @@ function generateLiveBlogPostingData (structuredData = {}, story = {}, publisher
 
 function generateVideoArticleData (structuredData = {}, story = {}, publisherConfig = {}) {
   const metaKeywords = story.seo && story.seo['meta-keywords'] || [];
+  const articleSection = story.sections && story.sections[0]['display-name'];
```
Comment (Jeevan-Kishore) [inline src/structured-data/structured-data-tags.js]: get

File: src/utils.js [hunk 5]
```diff
@@ -16,3 +16,12 @@ export function stripMillisecondsFromTime(date) {
     return toReturn;
   return toReturn.split('.')[0]+"Z";
 }
+
+export function getQueryParams(url) {
+  const urlObj =  new URL(url)
```
Comment (Jeevan-Kishore) [inline src/utils.js]: ;

**Pull request merged**